### PR TITLE
Changing S5 prints to logs

### PR DESCRIPTION
**Pull request by taylorbell57**
I changed almost all of the print statements in S5 into logs as per Kevin's request so that S5's outputs are all saved. The only print statements remaining are those before the log file is created.

### CODE REVIEW

**Review by kevin218 — DISMISSED** [review]

**Comment by taylorbell57** [comment]
Needed to resolve some merge conflicts with your edits Kevin, but now this is back and ready to merge once I have your updated approval

**Review by kevin218 — APPROVED** [review]

**Pull request merged**
